test/test_respondent_controller.py

```diff
@@ -311,6 +311,25 @@ def test_resend_verification_email_sends_via_notify(self):
             respondent.party_uuid
         )
 
+    def test_email_verification_expired_token_sends_calls_notify(self):
+        # Given a token is used that has already been declared to be expired
+        respondent = self.populate_with_respondent()
+        token = self.generate_valid_token_from_email(respondent.email_address)
+        # When the resend verification with expired token endpoint is hit
+        self.resend_verification_email_expired_token(token)
+        # Then a notification is sent the the respondent's email adddress
+        self.assertTrue(self.mock_notify.verify_email.called)
+
+    def test_resend_verification_email_expired_token_respondent_not_found(self):
+        # The token is valid but the respondent doesn't exist
+        current_app.config['EMAIL_TOKEN_EXPIRY'] = -1
+        token = self.generate_valid_token_from_email('invalid@email.com')
+        #When the resend verification with expired token endpoint is hit
```

Comment:
add a space here between `#` and `When` otherwise linting fail

Reply:
It did fail the linting.  Updated that now.